sphinx/sphinx.gradle

```diff
@@ -38,7 +49,7 @@ def sphinxImage = Docker.registerDockerImage(project, 'sphinx') {
     inputs.files copyPyClientWhl.get().outputs.files
     inputs.files copySphinxLib.get().outputs.files
     inputDir.set layout.buildDirectory.dir('sphinx-image')
-    inputs.files project(':docker-runtime-base').tasks.findByName('buildDocker').outputs.files // deephaven/runtime-base
+    inputs.files project(':docker-server').tasks.findByName('build').outputs.files // deephaven/docker/server
```

Comment:
We shouldn't depend on the `build` task here - it's doing a lot more than we need for deephaven/server (also, the comment is wrong). We should depend on the `buildDocker-server` task.